config/routes.rb

```diff
@@ -351,6 +351,7 @@
       collection do
         post 'add_existing_annotation'
         patch 'update_annotation'
+        post 'edit'
```

Comment:
You shouldn't need this. `resources :annotations` automatically creates an `edit` route, except it's a `GET` (which is the standard).